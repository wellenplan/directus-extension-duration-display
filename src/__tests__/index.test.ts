import index from "../index";

describe("duration", () => {
  describe("display", () => {
    describe("index", () => {
      it("should define a display", () => {
        expect(index).toMatchSnapshot();
      });
    });
  });
});
