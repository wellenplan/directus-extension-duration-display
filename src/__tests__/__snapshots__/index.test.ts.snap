// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`duration display index should define a display 1`] = `
Object {
  "component": Object {
    "props": Object {
      "value": Object {
        "required": false,
        "type": [Function],
      },
    },
    "render": [Function],
    "setup": [Function],
  },
  "description": "$:duration.display.description",
  "icon": "timer",
  "id": "duration",
  "name": "$:duration.display.name",
  "options": null,
  "types": Array [
    "integer",
    "float",
    "decimal",
  ],
}
`;
